Sources/System/Errno.swift

```diff
@@ -1377,12 +1395,13 @@ public struct Errno: RawRepresentable, Error, Hashable, Codable {
   @_alwaysEmitIntoClient
   @available(*, unavailable, renamed: "notSupportedOnSocket")
   public static var EOPNOTSUPP: Errno { notSupportedOnSocket }
```

Comment:
`EOPNOTSUPP` seems available https://github.com/WebAssembly/wasi-libc/blob/c5264e2bbe532994d06b039005f2af91bedcc1a6/expected/wasm32-wasi/predefined-macros.txt#L281

Reply:
Is that only available in a newer WASI SDK? It's apparently no there in `DEVELOPMENT-SNAPSHOT-2024-04-29-a-wasm32-unknown-wasi`:

```
swift-system/Sources/System/Internals/Constants.swift:447:34: error: cannot find 'EOPNOTSUPP' in scope
445 |
446 | @_alwaysEmitIntoClient
447 | internal var _EOPNOTSUPP: CInt { EOPNOTSUPP }
    |                                  `- error: cannot find 'EOPNOTSUPP' in scope
448 |
449 | #if SYSTEM_PACKAGE_DARWIN
```

Reply:
I guess you need shims for them to help ClangImporter